examples/on_gke_end_to_end/README.md

```diff
@@ -34,19 +34,21 @@ The [project factory](https://github.com/terraform-google-modules/terraform-goog
 - [Terraform Provider for GCP](https://www.terraform.io/docs/providers/google/index.html) v2.9
 
 ### Configure a Service Account
```

Comment:
This should have a different number of hashes (i.e. text should be bigger) than the section below it, or else it should be removed if it's not being used.

Reply:
Good catch.  That should be removed.